monitoring/README.md

```diff
@@ -92,47 +174,124 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
 a) API Server Monitoring: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-monitoring.png" width="60%" height="80%" title="apiserver-monitoring"> 
 
-To monitor a WDS space (e.g., wds1) use the following parameters: `NS: wds1-system` and `APIService: wds1`. Likewise, to monitor a ITS space (e.g., its1): `NS: its1-system` and `APIService: vcluster`
+To monitor a WDS space (e.g., wds1) use the following combination of `NS/APIService` parameters: `NS: wds1-system` and `APIService: wds1`. Likewise, to monitor a ITS space (e.g., its1): `NS: its1-system` and `APIService: vcluster`
+
+**Note**: 
+   - You can also use the following combination of `NS/APIService` values to monitor the APIserver of the hosting cluster or a WEC cluster: `NS: default` and `APIService: apiserver`
+   - Use the following commands to obtain the Ip address of the APIServer in the hosting cluster, WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces:
+     ```bash
+     kubectl --context <hosting-cluster-context> get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
+     ```
+     For example: 
+     ```bash
+     $ kubectl --context kind-kubeflex get Endpoints -A | egrep "NAME| wds| vcluster |kubernetes"
+     ```
+     Output:
+     ```bash
+     NAMESPACE         NAME                                                              ENDPOINTS                                                AGE
+     default           kubernetes                                                        172.18.0.2:6443                                          3d20h
+     its1-system       vcluster                                                          10.244.0.12:8443,10.244.0.12:8443                        3d20h
+     wds1-system       wds1                                                              10.244.0.21:9444                                         3d20h
+     ```
+     
+     Use the following command to obtain the IP address of the host container for your cluster: `kubectl --context <cluster-context> get Endpoints -n default`
+
+     For example: 
+     ```bash
+     kubectl --context cluster1 get Endpoints -n default 
+     ```
+
+     Output:
+     ```bash
+     NAME         ENDPOINTS         AGE
+     kubernetes   172.18.0.3:6443   4d14h
+     ```
+
+Here are the default port number for the APIservers: 
```

Comment:
I am not sure "default" is on point here. The reader needs to know how to determine the correct actual port number for their actual running system. The command exhibited just above does this for WEC apiservers. Other commands further above cover other rows in this table. Is this table needed?

Reply:
This table is just a helper table for the user. The information showed there are the typical information for an environment created using kind clusters.